### PR TITLE
Symfony5 support

### PR DESCRIPTION
**Pull request by Adrians-G**
Tests are green and it seems to be working fine locally. 

What could be other things we need to check?

### CODE REVIEW

**Comment by Adrians-G** [comment]
@Oliboy50 What are the next steps? Is there someone else who should take a look? 

How far are we to create a new tag? Thanks a lot!

**Comment by Oliboy50** [comment]
@Adrians-G could you please update the [`composer.json`](https://github.com/M6Web/DaemonBundle/blob/master/composer.json) file accordingly?

```json
        "symfony/console" : "~4.3 || ~5.0",
        "symfony/framework-bundle" : "~4.3 || ~5.0",
        "symfony/yaml": "~4.3 || ~5.0",
```

File: src/Command/DaemonCommand.php [hunk 1]
```diff
@@ -256,7 +256,7 @@ protected function loop(InputInterface $input, OutputInterface $output): void
             $this->dispatchEvent(DaemonLoopExceptionGeneralEvent::class);
 
             if ($this->getShowExceptions()) {
-                $this->getApplication()->renderException($e, $output);
```
Comment (omansour) [inline src/Command/DaemonCommand.php]: for me renderThrowable isnt available in sf4 ❓ 
so it is not retro compatible as the code seems to suggest.

File: composer.json [hunk 2]
```diff
@@ -12,9 +12,9 @@
         "php" : ">=7.2",
         "ext-pcntl" : "*",
         "psr/event-dispatcher": "^1.0",
-        "symfony/console" : "~4.3",
-        "symfony/framework-bundle" : "~4.3",
-        "symfony/yaml": "~4.3",
+        "symfony/console" : "~4.3|~5.0",
+        "symfony/framework-bundle" : "~4.3|~5.0",
+        "symfony/yaml": "~4.3|~5.0",
```
Comment (Oliboy50) [inline composer.json]:
```suggestion
        "symfony/yaml": "~4.3 || ~5.0",
```

File: composer.json [hunk 3]
```diff
@@ -12,9 +12,9 @@
         "php" : ">=7.2",
         "ext-pcntl" : "*",
         "psr/event-dispatcher": "^1.0",
-        "symfony/console" : "~4.3",
-        "symfony/framework-bundle" : "~4.3",
-        "symfony/yaml": "~4.3",
+        "symfony/console" : "~4.3|~5.0",
+        "symfony/framework-bundle" : "~4.3|~5.0",
```
Comment (Oliboy50) [inline composer.json]:
```suggestion
        "symfony/framework-bundle" : "~4.3 || ~5.0",
```

File: composer.json [hunk 4]
```diff
@@ -12,9 +12,9 @@
         "php" : ">=7.2",
         "ext-pcntl" : "*",
         "psr/event-dispatcher": "^1.0",
-        "symfony/console" : "~4.3",
-        "symfony/framework-bundle" : "~4.3",
-        "symfony/yaml": "~4.3",
+        "symfony/console" : "~4.3|~5.0",
```
Comment (Oliboy50) [inline composer.json]:
```suggestion
        "symfony/console" : "~4.3 || ~5.0",
```

File: src/Command/DaemonCommand.php (hunk 1 above)
Comment (Oliboy50) [reply src/Command/DaemonCommand.php]: `renderThrowable` has been introduced in symfony 4.4

we could either drop support for symfony 4.3 => `"symfony/console" : "~4.4 || ~5.0",`

or still support symfony 4.3 => 
```
if (method_exists(renderThrowable)) { renderThrowable } 
else { renderException }
```

Comment (Adrians-G) [reply src/Command/DaemonCommand.php]: Great catch!

Comment (Adrians-G) [reply src/Command/DaemonCommand.php]: Changed to 4.4

Comment (Adrians-G) [reply src/Command/DaemonCommand.php]: Dropped support for 4.3.

**Comment by Adrians-G** [comment]
@Oliboy50 Honestly, I have no idea how to make it run on Symfony 5 as the composer chooses one version. 

Maybe there can be another composer.json file, but that seems like a workaround.

File: .travis.yml [hunk 5]
```diff
@@ -3,9 +3,20 @@ language: php
...
+    - SYMFONY_VERSION="~4.4"
+    - SYMFONY_VERSION="~5.0"
+    
+before_install:
   - wget http://getcomposer.org/composer.phar
-  - php composer.phar install
+  - php composer.phar require "symfony/console:${SYMFONY_VERSION}" --no-update
+  - php composer.phar require "symfony/framework-bundle:${SYMFONY_VERSION}" --no-update
+  - php composer.phar require "symfony/yaml:${SYMFONY_VERSION}" --no-update
+  - php composer.phar require "symfony/console:${SYMFONY_VERSION}" --no-update
```
[the remark this answers is not in the document]
Comment (Adrians-G) [reply .travis.yml]: I will check today. I was experimenting.

**Comment by Adrians-G** [comment]
CI seems to be passing now. Please take a look for further suggestions.

**Comment by Oliboy50** [comment]
Thanks for your time 
=> [6.0.0](https://github.com/M6Web/DaemonBundle/releases/tag/v6.0.0) has been released

**Pull request merged**